disperser/apiserver/server.go

```diff
@@ -675,9 +712,11 @@ func (s *DispersalServer) RetrieveBlob(ctx context.Context, req *pb.RetrieveBlob
 			},
 		})
 		if err != nil {
+			s.metrics.HandleInternalFailureRpcRequest("RetrieveBlob")
 			return nil, api.NewInternalError(fmt.Sprintf("ratelimiter error: %v", err))
 		}
 		if !allowed {
+			s.metrics.HandleRateLimitedRpcRequest("RetrieveBlob")
 			s.metrics.HandleFailedRequest(codes.ResourceExhausted.String(), "", 0, "RetrieveBlob")
```

Comment:
Would we also remove this quorum specific metrics?

Reply:
I'd do it in followup, making the goal of this PR focused